### PR TITLE
Discover also TrustedFSPageTemplate files

### PR DESCRIPTION
**Pull request by jugmac00**
...  which are registered with the file ending .xpt.

More information on this kind of template file:
https://pypi.org/project/Products.TrustedExecutables/

modified:   CHANGES.rst
modified:   README.rst
modified:   src/gocept/template_rewrite/main.py
new file:   src/gocept/template_rewrite/tests/fixture/three.xpt
modified:   src/gocept/template_rewrite/tests/test_main.py

### CODE REVIEW

**Comment by jugmac00** [comment]
Hm, I can't request a reviewer - so... cc @icemac / @sallner

**Comment by jugmac00** [comment]
This fixes #6

File: CHANGES.rst [hunk 1]
```diff
@@ -5,4 +5,5 @@ Change log for gocept.template_rewrite
 0.1 (unreleased)
 ================
 
-initial release
+- File discovery now also includes TrustedFSPageTemplate files, which
+are registered with the file ending .xpt.
```
Comment (icemac) [inline CHANGES.rst]: ReST expects the second line indented:
```suggestion
  are registered with the file ending `.xpt`.
```

**Comment by icemac** [comment]
@jugmac00 You seem not have allowed edits from repository owners on your PR, so I am not able to commit my suggestion myself.

File: CHANGES.rst (hunk 1 above)
Comment (jugmac00) [reply CHANGES.rst]: Thank you for the review. I just installed a new rst-plugin for my editor - I still have not found a real good one.

**Comment by jugmac00** [comment]
> @jugmac00 You seem not have allowed edits from repository owners on your PR, so I am not able to commit my suggestion myself.

Until now I did not know such an option exits - but when looking to the right, the checkbox next to "Allow edits from maintainers." is checked.

**Comment by icemac** [comment]
@jugmac00 wrote:
> Until now I did not know such an option exits - but when looking to the right, the checkbox next to "Allow edits from maintainers." is checked.

That's strange. I got the error message that I am not allowed to commit my suggestion.

**Pull request merged**